### DISCUSSION TITLE
Where can I find the source code?

### DISCUSSION THREAD

**Discussion by hisengberg** [anujpathania/HotSniper#37] [General]
Dear Anuj,

I have the honor to read your paper 《Power-and Cache-Aware Task Mapping with Dynamic Power Budgeting for Many-Cores》, and in this paper, you said you will release the source code. I would like to know where can I get the source code in this paper？

Hope to get your reply soon, thank you very much.

-- Walter

![117631688453430_ pic](https://github.com/anujpathania/HotSniper/assets/118739669/ce6c3897-0712-484c-831a-b69a1bc479f2)
<img width="657" alt="image" src="https://github.com/anujpathania/HotSniper/assets/118739669/18ca5243-6aaa-46a0-a236-8b3bff4cbbfa">

**Comment by anujpathania** [comment]
Hi Walter,

The code was released as a scheduling policy on a separate branch on HotSniper - https://github.com/anujpathania/HotSniper/tree/IEEE-TC20-PCGov

I hope this helps.

-- Anuj

**Reply by hisengberg** [reply]
Hi Anuj,

Thank you for your help, Thank you so much.

-- Walter

**Reply by anujpathania** [reply]
Hi Walter,

If you are happy with the code, can you be kind enough to the close the discussion.

-- Anuj

**Reply by hisengberg** [reply]
Hi Anuj,

Yes, Sir. Thank you for your help again.

-- Walter
